spec/models/in_person_enrollment_spec.rb

```diff
@@ -491,4 +494,39 @@
       expect(failed_enrollment_without_notification.eligible_for_notification?).to eq(false)
     end
   end
+
+  describe 'enhanced_ipp?' do
+    before do
+      allow(IdentityConfig.store).to receive(:usps_eipp_sponsor_id).and_return(usps_eipp_sponsor_id)
+    end
+
+    context 'when IdentityConfig.store.usps_eipp_sponsor_id equals the enrollment sponsor_id' do
+      it 'returns true' do
+        user = create(:user)
+        profile = create(:profile, gpo_verification_pending_at: 1.day.ago, user: user)
+        enrollment = create(
+          :in_person_enrollment, user: user, profile: profile,
+                                 sponsor_id: usps_eipp_sponsor_id
+        )
+
+        expect(enrollment.enhanced_ipp?).to be true
+      end
+    end
+
+    context <<~STR.squish do
+      when IdentityConfig.store.usps_eipp_sponsor_id does
+       not equals the enrollment sponsor_id
+    STR
```

Comment:
I think with some minor rewording we could avoid the multiline description. What do you think of the following?

```suggestion
    context 'when the enrollment sponsor ID does not equal the EIPP sponsor ID' do
```

Reply:
Good suggestion! Resolved with commit 377af30457f2fed7acd3d0c10929b05e49529ff1